src/journal/journalctl.c

```diff
@@ -683,8 +683,9 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_VERIFY_KEY:
                         arg_action = ACTION_VERIFY;
-                        arg_verify_key = optarg;
+                        arg_verify_key = strdup(optarg);
```

Comment:
oom check is necessary.

Please also put
  `free(arg_verify_key)`
at end of `main`.